content/docs/getting_started/install_apps.md

```diff
@@ -105,30 +105,39 @@ In addition, a [Kubernetes Service Account](https://kubernetes.io/docs/tasks/con
 
 ### View the Application UIs
 
-Set up client port forwarding with the following steps to access the applications in the Kubernetes cluster. It is best to start a new terminal session for running the port forwarding script to maintain the port forwarding session, while using the original terminal to continue to issue commands. The port-forward-all.sh script will look for a `.env` file for environment variables needed to run the script. The `.env` creates the necessary variables that target the previously created namespaces. We will use the reference `.env.example` file and then run the port forwarding script.
+Set up client port forwarding with the following steps to access the applications in the Kubernetes cluster. It is best to start a new terminal session for running the port forwarding scripts to maintain the port forwarding session, while using the original terminal to continue to issue commands. The port-forward scripts will look for a `.env` file for environment variables needed to run the script. The `.env` creates the necessary variables that target the previously created namespaces. We will use the reference `.env.example` file and then run the port forwarding scripts.
 
 In a new terminal session, run the following commands to enable port forwarding into the Kubernetes cluster from the root of the project directory (your local clone of [upstream OSM](https://github.com/openservicemesh/osm)).
 
 ```bash
 cp .env.example .env
-./scripts/port-forward-all.sh
+bash <<EOF
+./scripts/port-forward-bookbuyer-ui.sh &
```

Comment:
We should consider removing the dependency on needing the OSM repo to run the demo. This seems to be a wrapper over the `kubectl port-forward` command, which can be invoked directly.